languages/go/exercises/concept/using-map/.docs/instructions.md

```diff
@@ -0,0 +1,22 @@
+In this excersie, you'll help uncle Donald to count monthly and total income of his thrift shop, he recorded of all of the transaction in his journal book, but he forgot to put the price it it, here's the price list
+
+| item name        | price |
+| ---------------- | ----- |
+| men_tops         | 2     |
+| men_underwares   | 1     |
+| men_jeans        | 4     |
+| robes            | 4     |
+| jackets          | 7     |
+| women_tops       | 4     |
+| women_underwares | 2     |
+| women_jeans      | 3     |
+
+## 1. Store the price table in your code, using map literal
+
+## 2. Count the monthly income
+
+Implement the `calculateMonthlyIncome` function, this function will count the amount that uncle get per month.
```

Comment:
Each task should have an example that shows what the return value of a function is for a specific input. For more information, see [this document](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd) and [this existing Go exercise](https://github.com/exercism/v3/blob/master/languages/go/exercises/concept/numbers/.docs/instructions.md#1-calculate-the-production-rate-per-hour). This comment also applies to the other tasks.